pkg/kube/secretcontroller/secretcontroller.go

```diff
@@ -219,14 +219,16 @@ func (c *Controller) Run(stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
 	defer c.queue.ShutDown()
 
+	t0 := time.Now()
 	log.Info("Starting Secrets controller")
 
 	go c.informer.Run(stopCh)
 
-	// Wait for the caches to be synced before starting workers
-	if !kube.WaitForCacheSyncInterval(stopCh, c.syncInterval, c.informer.HasSynced) {
+	if !cache.WaitForCacheSync(stopCh, c.informer.HasSynced) {
```

Comment:
can we use WaitForCacheSyncInterval? It makes our tests like 1min faster IIRC

Reply:
Sure. I do not remember why I changed this.